plugins/feeds/public/ViruSign.py

```diff
@@ -0,0 +1,64 @@
+import csv
+import logging
+import requests
+from datetime import datetime, timedelta
+from core.observables import Hash
+from core.feed import Feed
+from core.config.config import yeti_config
+from core.errors import ObservableValidationError
+
+class ViruSign(Feed):
+
+    default_values = {
+        "frequency": timedelta(hours=24),
+        "name": "ViruSign",
+        "source": "https://www.virusign.com/get_hashlist.php?ssdeep&imphash&sha256&sha1&md5&n=ANY&start_date={date}&end_date={date}",
+        "description": "This feed contains daily list of hashes from ViruSign",
+    }
+
+    def update(self):
+        today = datetime.now().strftime("%Y-%m-%d")
+        try:
+            r = requests.get(self.source.format(date=today), headers={"User-Agent": "yeti-project"})
+            if r.ok:
+                reader = csv.reader(r.content.splitlines(), quotechar='"')
+                for line in reader:
+                    self.analyze(line)
+        except Exception as e:
+            logging.error(e)
+
+    def analyze(self, line):
+        ssdeep, imphash, sha256, sha1, md5 = line
+        context = {}
+        context['date_added'] = datetime.now()
```

Comment:
Please don't include `datetime.now()` in context as it will create a new entry every time. If you really want to include it you should pass the field as a deduplication argument (see `add_context` docstring for details)